docs/Contribute/Agenda.md

```diff
@@ -0,0 +1,28 @@
+# Calls Calender.
+
+All calls, design, development, and Documentation (marketing, SEO) are open to everyone to join. 
+If you want to see the upcoming scheduled Calls, follow the guide below. 
+
+Calls are scheduled via Github issues. 
```

Comment:
Scheduling calls is irrelevant for this page. A user landing on this page should know how to join, and you should try to proiritize that ,scheduling is always done by people who already know the process, if you really want to document it, try positioning it after the "subscribing to calls"